styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,15 @@
-.footer{
-  
-}
\ No newline at end of file
+.rug{
+  .text37{
```

Comment:
I favor a more modular class name here like `.rug-header` or `.rug-text-large` that isn't nested.